### ISSUE TITLE
TauP

### ISSUE THREAD

**Issue by trac2github** [obspy/obspy#369]
Hi all,

There are two points about obspy.taup that I want to highlight here:

1) There is one parameter in the original taup software called "Pierce" that shows in which depth the ray pierce one specific discontinuity. It seems that it would be useful to have it here as well. 

2) in the original version of taup, we have the following travel-time tables:
1. iasp91
2. prem
3. ak135
4. qdt
while in the obspy version we just have number 1 and 3. Since many codes use PREM as the background model, is there any possibility to add it as well?

Thanks a lot in advance for your time and kind attention to my request.

Best,
Kasra

**Comment by AntonBiryukovUofC** [comment]
Out of curiousity: how difficult would there be to implement local velocity model ray path calculations based on the existing paradigm of TauP in obspy?

I find it challenging to find a software that could show raypaths in the local (shallow velocity model, taken from borehole logs for example). I have managed to squeeze the juice out of custom-made model functionality you guys implemented here, and it shows me the direct arrival times/ray-paths. That is already incredibly great and a big life-saver. However, adding the possibility to see the arrival times for the multiples (internal, free-surface) would be of incredible value, as well as the interface waves. I have noticed it works for the global earth-model, and therefore am curious whether there's a simple way to make interfaces in the local model seem like the "global" ones.

Thanks in advance,

Anton.
